### PR TITLE
fix(addCreateErrors): add errors for certain creations

### PR DESCRIPTION
**Pull request by KemoPaw**
https://satellite-im.atlassian.net/browse/AP-296

### CODE REVIEW

File: en-US.js [hunk 1]
```diff
@@ -394,6 +394,7 @@ export default {
       server_name_placeholder: 'Server name...',
       create_server: 'Create server',
       select_friends: 'Invite your friends to this server',
+      server_name_error: 'Server name must be at least 5 characters',
```
Comment (stavares843) [inline en-US.js]: maybe we can add a dot on the end 

the other similar errors also have a dot on the end

**Comment by stavares843** [comment]
thanks for adding! @KemoPaw 🎉 

minor suggestions above

File: en-US.js (hunk 1 above)
Comment (KemoPaw) [reply en-US.js]: @stavares843 Good idea, I must have forgotten!

**Comment by KemoPaw** [comment]
> thanks for adding! @KemoPaw 🎉
> 
> minor suggestions above

@stavares843 May I get another check once you have the time?

**Review by stavares843 — APPROVED** [review]

**Comment by stavares843** [comment]
thanks! 🎉

**Pull request merged**